src/base/digest32.h

```diff
@@ -41,13 +43,15 @@ class Digest32
     using UnderlyingType = lt::digest32<N>;
 
     Digest32() = default;
+    Digest32(const Digest32 &other) = default;
```

Comment:
also this: https://doc.qt.io/qt-5/qshareddatapointer.html#optimize-performance-for-usage-in-qt-containers

Reply:
> also this: https://doc.qt.io/qt-5/qshareddatapointer.html#optimize-performance-for-usage-in-qt-containers

I can't apply it to Digest32 since it is template.
Maybe apply it to TorrentID?

Reply:
>Maybe apply it to TorrentID?

I guess it can but it will need some in-code comments at `TorrentID` to assure that it is the base type enables it.

Reply:
> > Maybe apply it to TorrentID?
> 
> I guess it can but it will need some in-code comments at `TorrentID` to assure that it is the base type enables it.

Done.